packages/bits/src/animations/expand.ts

```diff
@@ -13,3 +15,21 @@ export const expand = trigger("expandedState", [
         animate("350ms ease-in-out"),
     ]),
 ]);
+
+    /**
+     * This animation is using the special selectors that ngIf and ngFor use on newly inserted or removed content.
+     */
```

Comment:
Indentation can be removed. Also, I would add something about why this animation is better. For example, "This v2 version of the expand animation respects the animations of elements nested inside the expander's content template."

One more thing, do you think we should deprecate the old expand animation? I think we maybe should do that. If so, it would be good to add a jsdoc comment like the following. And, then of course create the ticket for removal of the old animation to reference in the comment.
```
/** @deprecated As of Nova v11, use 'expandV2' instead. Removal: NUI-???? */
```